docs/designers-developers/developers/tutorials/javascript/js-build-setup.md

```diff
@@ -119,7 +119,7 @@ To configure npm to run a script, you use the scripts section in `package.json`
 
 You can then run the build using: `npm run build`.
 
-After the build finishes, you will see the built file created at `build/index.js`.
+After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as shown in Step 2, and the block will load in the editor.
```

Comment:
There is nothing labeled "Step 2". It would be best to reference something that exists/can be found on the current page.